gravitee-apim-console-webui/src/management/api/general/plans/list/api-general-plan-list.component.html

```diff
@@ -70,8 +70,13 @@
   <!-- Display Name Column -->
   <ng-container matColumnDef="name">
     <th mat-header-cell *matHeaderCellDef id="name">Name</th>
-    <td mat-cell *matCellDef="let element" class="plans__table__name" (click)="navigateToPlan(element.id)">
-      {{ element.name }}
+    <td mat-cell *matCellDef="let element" [class]="element.status !== 'CLOSED' ? 'plans__table__name' : ''">
```

Comment:
Instead of checking `!== 'CLOSED'` twice (once in the td and once in the div), you can check once in the td and have a second td with an ngIf for `element.status === 'CLOSED'`.